main.go

```diff
@@ -288,10 +288,18 @@ func downloadSourcePaths(sourcePaths []string, destPath string, githubRepo GitHu
 
 	// Unzip and move the files we need to our destination
 	for _, sourcePath := range sourcePaths {
-		fmt.Printf("Extracting files from <repo>%s to %s ...\n", sourcePath, destPath)
-		if err := extractFiles(localZipFilePath, sourcePath, destPath); err != nil {
+		fmt.Printf("Extracting files from <repo>%s to %s ... ", sourcePath, destPath)
+		if fileCount, err := extractFiles(localZipFilePath, sourcePath, destPath); err != nil {
+			fmt.Println()
```

Comment:
NIT: Might be worth printing how many files were extracted even if there was an error part of the way through...